climada/hazard/test/test_base_xarray.py

```diff
@@ -162,6 +159,29 @@ def test_type_and_unit(self):
         self.assertEqual(hazard.haz_type, "TC")
         self.assertEqual(hazard.units, "m/s")
 
+    def test_event_no_time(self):
+        """Test if an event coordinate that is not a time works"""
+        with xr.open_dataset(self.netcdf_path) as dataset:
+            size = dataset.sizes["time"]
+
+            # Integers
+            time = np.arange(size)
+            dataset["time"] = time
+            self.time = ["1970-01-01", "1970-01-02"]  # These will be 0, 1 as ordinals
```

Comment:
I think ordinals must be larger equal to 1. Is the comment thus incorrect?

Reply:
Yes, the test was wrong.